lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -290,9 +290,20 @@ class FlutterViewEmbedder {
         .prepareAccessibilityPlaceholder();
 
     glassPaneElementHostNode.nodes.addAll(<html.Node>[
-      semanticsHostElement,
       _accessibilityPlaceholder,
       _sceneHostElement!,
+
+      // The semantic host goes last because hit-test order-wise we want it to
```

Comment:
nit: avoid we

Reply:
here and everywhere, `we` may refer to different entity based on the reader's perspective.

Reply:
Done.